tests/test_theseus_layer.py

```diff
@@ -85,7 +85,7 @@ def create_qf_theseus_layer(
     cost_weight=th.ScaleCostWeight(1.0),
     nonlinear_optimizer_cls=th.GaussNewton,
     linear_solver_cls=th.CholeskyDenseSolver,
-    max_iterations=10,
+    max_iterations=50,
```

Comment:
Why change this default?

Reply:
Yeah, better to set the value in the function call rather than change the default (which might silently affect a lot of other code).

Reply:
make sense, will revert